services/frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
```

Comment:
Thoughts on calling it `@logan/web-frontend` for consistency?

Reply:
Fixing it now, that was an error for the script used to generate the file.